lib/provider/volume_mounter.go

```diff
@@ -76,6 +78,9 @@ type VolumeResponse struct {
 	// By default we assume all the capabilities are supported.
 	// If the plugin does not support a capability, it can return false for that capability.
 	Capabilities map[string]bool `json:"capabilities,omitempty"`
```

Comment:
why `Capabilities ` , `VolumeName` , `DevicePath` is required?

Reply:
Will move Mount related code to CSI driver